src/Controller/TrickController.php

```diff
@@ -3,103 +3,190 @@
 namespace App\Controller;
 
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
-use Symfony\Component\HttpFoundation\Request;
 use Doctrine\ORM\EntityManagerInterface;
+
+use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
+
+use App\Service\TrickManager;
+use App\Service\FileUploader;
 
 use App\Entity\Trick;
+use App\Entity\Video;
 use App\Entity\Comment;
+use App\Entity\Image;
+
 use App\Repository\TrickRepository;
 use App\Repository\CommentRepository;
+use App\Repository\VideoRepository;
+
 use App\Form\TrickType;
 use App\Form\CommentType;
 
+use Symfony\Component\HttpFoundation\File\File;
+use Doctrine\Common\Collections\ArrayCollection;
+
 class TrickController extends AbstractController
 {
-    #[Route('/tricks', name: 'tricks')]
-    public function getAllTricks(TrickRepository $trickRepository)
+    #[Route('/', name: 'home')]
+    public function getAllTricks(TrickRepository $trickRepository, Request $request)
     {
         $tricks = $trickRepository->findAll();
+        $user = $this->getUser();
 
-        return $this->render('pages/tricks/index.html.twig', [
-            'tricks' => $tricks
+        return $this->render('pages/home/index.html.twig', [
+            'tricks' => $tricks,
+            'user' => $user
```

Comment:
user is accessible in twig from app.user, you don't need to send it in the params